src/aluno/CadastroAlunoPage.tsx

```
@@ -63,19 +56,36 @@ export function CadastroAlunoPage() {
       registroAcademico: '',
       id: undefined,
     },
   })
 
-  function onSubmit(values: z.infer<typeof formSchema>) {
-    toast.info('Cadastrando aluno...')
+  const { reset } = form
+
+  useEffect(() => {
+    if (location.state?.email && location.state?.code) {
+      setEmail( location.state.email)
```

ESLint

[error] Delete `·`

(prettier/prettier)